packages/mds-compliance-api/logger.ts

```diff
@@ -0,0 +1,3 @@
+import logger from '@mds-core/mds-logger'
+
+export const ComplianceApiLogger = logger.createLogger('ComplianceApiLogger')
```

Comment:
I don't think you need `Logger` in the namespace name. Again, I am a fan of using the package name as the namespace to associate the logs with their corresponding modules.